sycl/include/sycl/detail/pi.def

```diff
@@ -145,4 +145,8 @@ _PI_API(piextUSMEnqueueFill2D)
 _PI_API(piextUSMEnqueueMemset2D)
 _PI_API(piextUSMEnqueueMemcpy2D)
 
+// Device global variable
+_PI_API(piextEnqueueDeviceGlobalVariableWrite)
+_PI_API(piextEnqueueDeviceGlobalVariableRead)
```

Comment:
Could we instead define an API that returns the address of the variable, which can be then used in existing memory copy interfaces? Note, we already defined `piextGetDeviceFunctionPointer`

Reply:
For both CUDA and L0 that would definitely make sense, but [cl_intel_global_variable_access](https://github.com/intel/llvm/blob/sycl/sycl/doc/design/opencl-extensions/cl_intel_global_variable_access.asciidoc) defines interfaces similar to these and as such we would need to somehow abstract the address such an API would return, then identify and handle that correctly in our memory functions, without incorrectly colliding with actual memory addresses. A case could be made for adding that in another OpenCL extension, but as it stands I don't believe we have a good way of accomplishing it with what we currently have.

Reply:
Makes sense to me.